modules/raft/src/test/java/org/apache/ignite/internal/raft/RaftGroupServiceTest.java

```diff
@@ -171,40 +156,30 @@ public void testRefreshLeaderNotElected() throws Exception {
         leader = null;
 
         RaftGroupService service =
-            RaftGroupServiceImpl.start(TEST_GRP, cluster, FACTORY, TIMEOUT, NODES, false, DELAY, executor).get(3, TimeUnit.SECONDS);
+                RaftGroupServiceImpl.start(TEST_GRP, cluster, FACTORY, TIMEOUT, NODES, false, DELAY, executor).get(3, TimeUnit.SECONDS);
 
         assertNull(service.leader());
 
         try {
             service.refreshLeader().get();
 
             fail("Should fail");
-        }
-        catch (ExecutionException e) {
+        } catch (ExecutionException e) {
             assertTrue(e.getCause() instanceof TimeoutException);
```

Comment:
How about changing this to `assertThat(e.getCause(), is(instanceOf(TimeoutException.class)))`? It will allow us to see the real exception type if it does not match our expectation